inst/mappings/mapping_NAVIGATE.csv

```diff
@@ -169,25 +169,25 @@ idx;Tier;Category;variable;unit;piam_variable;piam_unit;piam_factor;internal_com
 132;2;demography;Population|Rural;million;;;;;;Total population living in rural areas;
 133;2;demography;Population|Urban;million;Population|Urban;million;;;;Total population living in urban areas;S
 134;2;demography;Population|Working Age;million;;;;;;Total working age population (age 15-65 years);
-135;1;economy;Consumption;billion US$2010/yr;Consumption;billion US$2005/yr;1.12;;;total consumption of all goods, by all consumers in a region;R
-136;1;economy;GDP|MER;billion US$2010/yr;GDP|MER;billion US$2005/yr;1.12;;;GDP at market exchange rate;R
-137;1;economy;GDP|PPP;billion US$2010/yr;GDP|PPP;billion US$2005/yr;1.12;;;GDP converted to International $ using purchasing power parity (PPP);R
+135;1;economy;Consumption;billion US$2010/yr;Consumption;billion US$2005/yr;1,12;;;total consumption of all goods, by all consumers in a region;R
+136;1;economy;GDP|MER;billion US$2010/yr;GDP|MER;billion US$2005/yr;1,12;;;GDP at market exchange rate;R
+137;1;economy;GDP|PPP;billion US$2010/yr;GDP|PPP;billion US$2005/yr;1,12;;;GDP converted to International $ using purchasing power parity (PPP);R
 138;1;economy;Value Added|Agriculture;billion US$2010/yr;;;;;;value added of the agricultural sector;
 139;1;economy;Value Added|Commercial;billion US$2010/yr;;;;;;value added of the commercial sector;
 140;1;economy;Value Added|Industry;billion US$2010/yr;;;;;;value added of the industry sector;
-141;1;economy;Capital Formation;billion US$2010/yr;Investments;billion US$2005/yr;1.12;;;net additions to the physical capital stock;R
+141;1;economy;Capital Formation;billion US$2010/yr;Investments;billion US$2005/yr;1,12;;;net additions to the physical capital stock;R
 142;1;economy;Expenditure|Government;billion US$2010/yr;;;;;;total government expenditure;
 143;1;economy;Expenditure|Household;billion US$2010/yr;;;;;;Total household expenditure;
 144;1;economy;Export;billion US$2010/yr;;;;;;total exports measured in monetary quantities.;
 145;1;economy;Import;billion US$2010/yr;;;;;;total imports measured in monetary quantities.;
 146;2;economy;Capital Stock;billion US$2010/yr;;;;;;Macroeconomic capital stock;
 147;2;economy;Interest Rate|Real;%;Interest Rate t/(t-1)|Real;unitless;100;;;Real interest rate or return on capital that is relevant for energy system investments;R
-148;2;economy;Value Added|Industry|Chemicals;billion US$2010/yr;Value Added|Industry|Chemicals;billion US$2005/yr;1.12;;;value added of the chemical industries.;R
```

Comment:
Can you please undo all the adjustments of `1.12` to `1,12` with a comma? This small script should do it:
```
devtools::load_all(); library(tidyverse)
write.csv2(getMapping("NAVIGATE") %>% mutate(piam_factor = gsub(",", ".", .data$piam_factor)), mappingNames("NAVIGATE"), na = "", row.names = FALSE, quote = FALSE)
```